alkos/kernel/src/kernel.cpp

```diff
@@ -11,21 +11,24 @@
 #endif
 
 #include <init.hpp>
+#include "utils/serial/serial64.h"
```

Comment:
I think this should be moves do drivers directory to qemu-serial-port subdirectory or something like that